content-src/components/ContextMenu/ContextMenu.jsx

```diff
@@ -138,15 +137,22 @@ export class ContextMenuItem extends React.PureComponent {
     }
   }
 
+  onKeyUp(event) {
+    if (event.key === " ") {
+      event.preventDefault();
+    }
+  }
+
```

Comment:
@AllegroFox can you please help clarify why we need onKeyUp handling here. Thanks